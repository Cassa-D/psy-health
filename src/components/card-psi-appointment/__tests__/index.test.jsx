import React from "react";
import renderer from "react-test-renderer";

import CardPsiAppointment from "../index";

const appointmentData = {
  userId: 14,
  id: 1,
  date: {
    start: "2020-09-07 13:00:00",
    end: "2020-09-07 14:00:00",
  },
  patient: {
    name: "John Fernandez",
    id: 14,
  },
  psic: {
    name: "Joana Camargo Mota",
    id: 1,
  },
};

describe("Render tests", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<CardPsiAppointment type="past" appointmentData={appointmentData} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
