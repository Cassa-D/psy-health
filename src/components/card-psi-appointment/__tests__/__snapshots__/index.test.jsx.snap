// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render tests renders correctly 1`] = `
<div
  className="sc-bdnylx knEsSx card-psi-appointment"
>
  <div
    className="ant-row"
    style={Object {}}
  >
    <div
      className="ant-col ant-col-xs-10 ant-col-sm-10 ant-col-md-10 ant-col-lg-10 ant-col-xl-10"
      style={Object {}}
    >
      <div
        className="col-container"
      >
        <div
          className="card-title"
        >
          John Fernandez
        </div>
      </div>
    </div>
    <div
      className="ant-col ant-col-xs-8 ant-col-sm-8 ant-col-md-8 ant-col-lg-8 ant-col-xl-8"
      style={Object {}}
    >
      <div
        className="col-container"
      >
        <div
          className="card-title"
        >
          Data da consulta
        </div>
        <div>
          07-09-2020
        </div>
      </div>
    </div>
    <div
      className="ant-col ant-col-xs-6 ant-col-sm-6 ant-col-md-6 ant-col-lg-6 ant-col-xl-6"
      style={Object {}}
    >
      <div
        className="col-container"
      >
        <div
          className="card-title"
        >
          Hor√°rio
        </div>
        <div>
          13:00
        </div>
      </div>
    </div>
  </div>
</div>
`;
